mods/beds/functions.lua

```diff
@@ -224,6 +238,37 @@ minetest.register_on_leaveplayer(function(player)
 	end
 end)
 
+minetest.register_on_dieplayer(function(player)
+	local pos = player:get_pos()
+	local yaw = get_look_yaw(pos)
+
+	lay_down(player, nil, pos, false)
+	player:set_look_horizontal(yaw)
+	player:set_pos(pos)
+
+	minetest.chat_send_all(player:get_player_name().." died.")
+end)
+
+function minetest.calculate_knockback(player, hitter, time_from_last_punch, tool_capabilities, dir, distance, damage)
```

Comment:
Do not copy the original implementation when overriding functions like this.

Reply:
The arguments or the code for the velocity when the player is hit?

Reply:
The code of the original implementation, you should call the existing function instead.

Reply:
Done!